### DISCUSSION TITLE
ScriptReference/ContextMenuItemAttribute

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#57] [Notes]
# ScriptReference/ContextMenuItemAttribute

https://docs.unity3d.com/ScriptReference/ContextMenuItemAttribute.html

**Comment by vertxxyz** [comment]
If you're looking to create a context menu item for any property you can use [`EditorApplication.contextualPropertyMenu`](https://docs.unity3d.com/ScriptReference/EditorApplication-contextualPropertyMenu.html).
